include/dev/apic.h

```diff
@@ -249,16 +249,24 @@ struct apic_dev {
 static inline void _apic_msr_write(uint32_t msr, 
 				   uint64_t data)
 {
+    // RISCV HACK
```

Comment:
is this still included in riscv code? Would it be better to fix the root abstraction than have this hack?